StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFlowController.swift

```diff
@@ -48,6 +48,22 @@ extension PaymentSheet {
                 return paymentMethod
             }
         }
+
+        // Both "Link" and "Instant Debits" use the same payment method type
+        // of "link." To differentiate between the two in metrics, we sometimes
+        // need a "link_context."
+        var linkContext: String? {
+            if case .link = self {
+                return "wallet"
+            } else if
+                case .new(let confirmParams) = self,
+                confirmParams.instantDebitsLinkedBank != nil
+            {
+                return "instant_debits"
+            } else {
+                return nil
+            }
+        }
```

Comment:
this tries to imitate what Android already comitted:

https://github.com/stripe/stripe-android/pull/8382/files#diff-25047a570b80aaccb7028233681fb7213439cb9b2113f21d57b0aa6e1840774aR564-R569